flask_project/campaign_manager/models/models.py

```diff
@@ -307,6 +324,34 @@ def get_participants(self):
         campaign."""
         return len(self.users)
 
+    def get_task_boundary_coordinates(self):
+        """ Returns the coordinates of the TaskBoundary. """
+        coordinates = self.get_task_boundary_as_geoJSON()
+        coordinates = ast.literal_eval(coordinates[0])
+        coordinates = coordinates['coordinates'][0]
+        return coordinates
+
+    def get_participant_count(uuid):
```

Comment:
I would love to see a test against this method.
What if a manager of a campaign belongs to a team of this same campaign? Do we count this user twice?

Reply:
can we use an old SQL query to get the participant count?

Reply:
You are probably right, I might have to correct that.

Reply:
I would update this test with the Crud tests branch as it contains all the tests for campaign methods.
The tests in the PR are outdated and are simple session.query() tests.